tests/jballerina-unit-test/src/test/java/org/ballerinalang/test/statements/matchstmt/varbindingpatternmatchpattern/ErrorBindingPatternTest.java

```diff
@@ -48,6 +49,8 @@ public void setup() {
                 "/error_binding_pattern_with_rest_binding_pattern.bal");
         resultNegative = BCompileUtil.compile("test-src/statements/matchstmt/varbindingpatternmatchpattern" +
                 "/error_binding_pattern_negative.bal");
+        resultNegative2 = BCompileUtil.compile("test-src/statements/matchstmt/varbindingpatternmatchpattern" +
```

Comment:
This test class is for match right? Shall we move this where appropriate instead?